delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/ce/CatsResponseToJsonLd.scala

```diff
@@ -0,0 +1,215 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.ce
+
+import akka.http.scaladsl.marshalling.ToEntityMarshaller
+import akka.http.scaladsl.model.MediaTypes.`application/json`
+import akka.http.scaladsl.model.StatusCodes.OK
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.{Accept, RawHeader}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import cats.effect.IO
+import cats.syntax.all._
+import ch.epfl.bluebrain.nexus.delta.kernel.effect.migration._
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfMediaTypes._
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.api.{JsonLdApi, JsonLdJavaApi}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.RemoteContextResolution
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.encoder.JsonLdEncoder
+import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonKeyOrdering
+import ch.epfl.bluebrain.nexus.delta.sdk.JsonLdValue
+import ch.epfl.bluebrain.nexus.delta.sdk.ce.CatsResponseToJsonLd.{RejOrFailOrComplete, UseLeft, UseRight}
+import ch.epfl.bluebrain.nexus.delta.sdk.ce.DeltaDirectives.{emit, jsonLdFormatOrReject, mediaTypes, requestMediaType, unacceptedMediaTypeRejection}
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.Response.{Complete, Reject}
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.{FileResponse, Response}
+import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.HttpResponseFields
+import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.JsonLdFormat.{Compacted, Expanded}
+import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.RdfMarshalling._
+
+import java.nio.charset.StandardCharsets
+import java.util.Base64
+
+sealed trait CatsResponseToJsonLd {
```

Comment:
To serialize both success and error channels in the API
Adapted from `ResponseToJsonLd`